packages/p2p/src/stores/order-store.js

```diff
@@ -199,7 +199,7 @@ export default class OrderStore {
                     const { p2p_order_list } = response;
 
                     this.root_store.general_store.handleNotifications(this.orders, p2p_order_list.list);
-                    p2p_order_list.list.forEach(order => this.syncOrder(order));
+                    p2p_order_list.list?.forEach(order => this.syncOrder(order));
                     this.setOrders(p2p_order_list.list);
```

Comment:
```typescript
const { list } = p2p_order_list || {};

if (list) {
     this.root_store.general_store.handleNotifications(this.orders, list);
     list.forEach(order => this.syncOrder(order));
     this.setOrders(list);
}
```

Reply:
`const { list } = p2p_order_list || {}` is a great suggestion! `if (list && list.length > 0)` can be rewritten as `if (list?.length)` instead 👍